antivir/scanners/scanner_clamav.py

```diff
@@ -58,5 +56,5 @@ def run(stream):
         """
         cd = ClamdAgnostic()
         scan_stream = cd.scan_stream(stream)
```

Comment:
scan_stream = ClamdAgnostic().scan_stream(stream)

Reply:
I agree with you.